dev-utils/visualise-ui-hierarchy/Makefile

```diff
@@ -0,0 +1,46 @@
+CWD := $(shell pwd)
+POLYPOD_FEATURES_PATH := $(CWD)/../../features
+RCH := node $(CWD)/react-component-hierarchy/bin/rch.js -e
+
+.PHONY: all
+all: polyExplorer.svg facebookImport.svg
+
+.PHONY: rebuild
+rebuild: clean-files all
+
+%.svg: %.dot
+	fdp -T svg -o $@ $<
+
+.INTERMEDIATE: %.dot
+%.dot: %.txt
```

Comment:
Configuration in the environment principle would say this needs another "deps-install" target to install tree2dot.

Reply:
Why? That script is lying next to the Makefile

Reply:
It's always better to have a clear target in the Makefile than to have to read through README, INSTALL and all that (and/or error messages when it does not find something)
I thought that was part of the graphviz package, ahyhow. I meant that you would need some target to install Python, graphviz and other stuff needed to run this.

Reply:
Ah, no I think that's totally overkill, instructions are in the README... Cloning some weird repo is one thing, quickly installing Graphviz via `brew` or similar isn't in my book. If this was production code I'd agree with you